data/json/items/magazines.json

```diff
@@ -0,0 +1,83 @@
+[
+    {   "type" : "MAGAZINE",
+        "id" : "stanag30",
+        "price" : 8000,
+        "name" : "STANAG magazine",
+        "symbol" : "#",
+        "color" : "grey",
```

Comment:
I'm sure your debug.log contains complains about that spelling. Please replace it by "light_gray"